addins/shortcut-sample-with-taskpane-CF/manifest.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Created: eec6a18b-71fe-4fdc-bba8-4f3ea6c92bd5 -->
+<OfficeApp 
+    xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0" 
+    xmlns:ov="http://schemas.microsoft.com/office/taskpaneappversionoverrides" 
+    xsi:type="TaskPaneApp">
+
+  <!-- Begin Basic Settings: Add-in metadata, used for all versions of Office unless override provided. -->
+
+  <!-- IMPORTANT! Id must be unique for your add-in, if you reuse this manifest ensure that you change this id to a new GUID. -->
+  <Id>eec6a18b-71fe-4fdc-bba8-4f3ea6c92bd5</Id>
+
+  <!--Version. Updates from the store only get triggered if there is a version change. -->
+  <Version>1.0.0.0</Version>
+  <ProviderName>Microsoft</ProviderName>
+  <DefaultLocale>en-US</DefaultLocale>
+
+  <!-- The display name of your add-in. Used on the store and various places of the Office UI such as the add-ins dialog. -->
+  <DisplayName DefaultValue="Shortcut Sample with taskpane and CF" />
+  <Description DefaultValue="Shortcut Sample with taskpane and CF" />
+
+  <!-- Icon for your add-in. Used on installation screens and the add-ins dialog. -->
+  <IconUrl DefaultValue="https://officedev.github.io/testing-assets/addins/shortcut-sample/images/button32x32.png" />
```

Comment:
We could use the shared one. `https://officedev.github.io/testing-assets/addins/images/button32x32.png`. And elsewhere.

Reply:
Fixed